build.gradle

```diff
@@ -56,7 +56,7 @@ subprojects {
         }
     }
 
-    String[] allowAndroidTestsIn = ["app"]
+    String[] allowAndroidTestsIn = ["app", "sync-lib"]
```

Comment:
Notice I've introduced an exception to have android tests on the sync-lib module. When we move the submodule to it's own repo, this will be back to only `:app`.